CONTRIBUTING.md

```diff
@@ -13,10 +13,36 @@ Some links and potential special rules for this repo only are listed below.
 * Wiki to translate: [wiki](../../wiki)
 * Sidebar file for adding language: [`_Sidebar` file](../../wiki/_Sidebar/_edit)
 
-### Coding guidelines
+## Coding guidelines
 
 * [editorconfig file](.editorconfig)
 
+### Adding symbol replacements
+
+If you want to add symbols to be replaced, do it like this:
+
+Add the item to the `symbols` object in the `src/common/modules/data/Symbols.js` file.
+The syntax is a simple `”key”: "value”` replacement.
```

Comment:
I think you wanted all ASCII quotes here: `”key”: "value”` → `"key": "value"`. Also below.